### PR TITLE
implement schema conversion interface

### PR DESCRIPTION
**Pull request by Anna050689**
implement schema conversion interface

### CODE REVIEW

File: src/syngen/ml/interface/interface.py [hunk 1]
```diff
@@ -139,7 +139,10 @@ def run(
         data, schema = DataLoader(source).load_data()
         # remove completely empty columns
         data = data.dropna(how="all", axis=1)
-        schema = {column: data_type for column, data_type in schema.items() if column in data.columns}
+        schema["fields"] = {
+            column: data_type for column, data_type in schema.get("fields", {}).items() if column in data.columns
```
Comment (kapikapi) [inline src/syngen/ml/interface/interface.py:143]: Hi! Will this work if `schema` is None? Cause None is returned for csv

Comment (Anna050689) [reply src/syngen/ml/interface/interface.py:143]: Thank you) I'll double-check it. Seems that you are right. In this case, we will have the exception.

Comment (kapikapi) [reply src/syngen/ml/interface/interface.py]: Thanks for checking! It fails in current version of main as well, so it would be great to fix it here :)

**Review by pbobyrev — APPROVED** [review]

File: src/syngen/ml/data_loaders/data_loaders.py [hunk 2]
```diff
@@ -63,10 +65,15 @@ class CSVLoader(BaseDataLoader):
     """
 
     @staticmethod
-    def _load_data(path, **kwargs) -> tuple[pd.DataFrame, None]:
-        df = pd.read_csv(path, **kwargs).iloc[:, :]
-        df.columns = df.columns.str.replace(':', '')
-        return df, None
+    def _load_data(path, **kwargs) -> Tuple[pd.DataFrame, None]:
+        try:
```
[the remark this answers is not in the document]
Comment (Anna050689) [reply src/syngen/ml/data_loaders/data_loaders.py:69]: Thanks for your comment. I'll fix it and wrap the "df.columns=" in finally block

**Pull request merged**